{
    "config": {
        "abort": {
            "already_configured": "Az eszk\u00f6z m\u00e1r konfigur\u00e1lva van",
            "cannot_connect": "Nem lehet csatlakozni a felfedezett Volumi\u00f3hoz"
        },
        "error": {
            "cannot_connect": "Sikertelen csatlakoz\u00e1s",
            "unknown": "V\u00e1ratlan hiba t\u00f6rt\u00e9nt"
        },
        "step": {
            "discovery_confirm": {
                "description": "Szeretn\u00e9d hozz\u00e1adni a Volumio (`{name}`)-t a Home Assistant-hoz?",
                "title": "Felfedezett Volumio"
            },
            "user": {
                "data": {
                    "host": "Hoszt",
                    "port": "Port"
                }
            }
        }
    }
}